docs/pages/modules/module-api.mdx

```diff
@@ -152,6 +152,26 @@ async function getMessageAsync() {
 }
 ```
 
+<hr />
+
+#### Kotlin coroutines <PlatformTags prefix="" platforms={['android']} />
+
+`AsyncFunction` can receive a suspendable body on Android. However, it has to be passed in the infix notation after the `Coroutine` block. You can read more about suspendable functions and coroutines on [coroutine overview](https://kotlinlang.org/docs/coroutines-overview.html).
+
+`AsyncFunction` with suspendable body can't receive `Promise` as an argument. It uses a suspension mechanism to execute asynchronous calls.
+The function is immediately resolved with the returned value of the provided suspendable block or rejected if it throws an exception. The function can receive up to 8 arguments.
+
+By default, suspend functions are dispatched on the module's coroutine scope. Moreover, every other suspendable function called from the body block are run on the same scope.
```

Comment:
"every other suspendable function" sounds like a singular 😉
Also "within scope" or "in scope" seem better than "on scope"

```suggestion
By default, suspend functions are dispatched on the module's coroutine scope. Moreover, every other suspendable function called from the body block is run within the same scope.
```